build-images.sh

```diff
@@ -12,6 +12,12 @@ repobase="${REPOBASE:-ghcr.io/nethserver}"
 #Get WebTop version
 webtop_version=$(cat ${PWD}/webtop5-build/VERSION)
 
+# Download of external deps and CHECKSUM verification:
+if [[ ! -f pecbridge-5.4.0.tar.gz ]]; then
```

Comment:
A note: the pecbridge file name is repeater 3 times, this could bring to some headache when upgrading it.
Is it possible to have it on a single place?